src/parser.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include <iomanip>
+#include <sstream>
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <dcmtk/dcmpstat/dcmpstat.h>
+#include "Parser.hpp"
+
+using json = nlohmann::json;
+
+using namespace std;
+
+
+/*! \The Parser class parses the configuration file with the user settings. 
+    \It includes basic functionalities for the moment. 
+*/
+void Parser::initial_settings(){
+
+    
+    auto js = R"({"GCC_compiler_version":"9.3.0",
+    "warn_return_any":"True",
+    "warn_unused_configs":"True",
+    "ignore_missing_imports":"True",
+    "include_private_tags":"True"})"_json; /**< create a string containing configuration files and convert it to a json object */
```

Comment:
Would it be better to move the test json to MyTests.cpp and have it given as input to the parser?

Reply:
Yes we want to give the test json as input to the parser. The stringstream from c++ did not integrated well with the json library as opposed to the file stream so I used temporarily raw string literals.